study-builder/studies/shared/family-history/family-history-top-level.conf

```diff
@@ -0,0 +1,314 @@
+  // Needs to be updated
+  "displayOrder": 1,
+  "writeOnce": false,
+  "maxInstancesPerUser": 1,
+  INSTRUCTIONS_BLOCK_SELF_EN={include required("snippets/family-history-section-instructions.conf")
+    INSTRUCTIONS_P1_EN="""
+              In this survey we would like to know the <span class="Semibold">living status, age, and cancer history</span> of people in your <span class="Semibold">biological, or blood-related, family.</span> We recognize that there are many different types of families, so please skip sections that do not apply to your family tree.""",
+    SHOWN_EXPR="""!user.studies[""""${id.study}""""].isGovernedParticipant()"""
+  },
+  INSTRUCTIONS_BLOCK_CHILD_EN={include required("snippets/family-history-section-instructions.conf")
+    INSTRUCTIONS_P1_EN="""
+              In this survey we would like to know the <span class="Semibold">living status, age, and cancer history</span> of people in <span class="Semibold">your child's biological, or blood-related, family.</span> We recognize that there are many different types of families, so please skip sections that do not apply to your family tree.""",
+    SHOWN_EXPR="""user.studies[""""${id.study}""""].isGovernedParticipant()"""
+  },
+  PARENT_SECTION_INTRO_BLOCK = {
+    include required("snippets/family-history-section-family-member-intro-block.conf")
+    SECTION_TITLE_EN = "Parents"
+  },
+  SIBLINGS_INTRO_BLOCK = {
+    include required("snippets/family-history-section-family-member-intro-block.conf")
+    SECTION_TITLE_EN = "Siblings & Half-Siblings"
+
+  },
+  CHILDREN_SECTION_INTRO_BLOCK = {
+    include required("snippets/family-history-section-family-member-intro-block.conf")
+    SECTION_TITLE_EN = Children
+  },
+  AUNT_UNCLE_SECTION_INTRO_BLOCK = {
+    include required("snippets/family-history-section-family-member-intro-block.conf")
+    SECTION_TITLE_EN = "Aunts & Uncles"
+  },
+  GRANDPARENTS_SECTION_INTRO_BLOCK = {
+    include required("snippets/family-history-section-family-member-intro-block.conf")
+    SECTION_TITLE_EN = "Grandparents"
+  },
+  ABOUT_YOU_SECTION_INTRO_BLOCK = {
+    include required("snippets/family-history-section-family-member-intro-block.conf")
+    SECTION_TITLE_EN = "About You"
+    FH_MEMBER_P1 = ""
+  },
+  "translatedNames": [
+    {
+      "language": "en",
+      "text": "Family History of Cancer"
+    }
+  ],
+  "translatedTitles": [
+    {
+      "language": "en",
+      "text": "Family History of Cancer"
+    }
+  ],
+  "translatedSubtitles": [
+    {
+      "language": "en",
+      "text": ""
+    }
+  ],
+  "translatedSummaries": [
+    {
+      "statusCode": "CREATED",
+      "language": "en",
+      // Needs to be updated
+      "text": """Completing the Family History of Cancer will help us understanding what role your genetics have had in your disease.."""
+    },
+    {
+      "statusCode": "IN_PROGRESS",
+      "language": "en",
+      // Needs to be updated
+      "text": """Family history in progress!!!."""
```

Comment:
Can probably make this say the same thing as the CREATED status one (btw there's an extra period at the end for both CREATED and IN_PROGRESS. if study staff doesn't have any input here, maybe we just remove these and NOT  have these status summaries?